apps/admin/backend/src/app.write_ins.test.ts

```diff
@@ -337,6 +345,67 @@ test('getWriteInImageView', async () => {
   `);
 });
 
+test('getWriteInImageView on bmd', async () => {
+  const { auth, apiClient } = buildTestEnvironment();
+  const { electionDefinition, castVoteRecordExport } =
+    electionTwoPartyPrimaryFixtures;
+  await configureMachine(apiClient, auth, electionDefinition);
+
+  const reportDirectoryPath = castVoteRecordExport.asDirectoryPath();
+  (
+    await apiClient.addCastVoteRecordFile({
+      path: reportDirectoryPath,
+    })
+  ).unsafeUnwrap();
+
+  // look at a contest that can have multiple write-ins per ballot
+  const contestId = 'zoo-council-mammal';
+  const writeInIds = await apiClient.getWriteInAdjudicationQueue({
+    contestId,
+  });
+  expect(writeInIds).toHaveLength(24);
+  const [writeInIdA, writeInIdB] = writeInIds.slice(0, 2);
```

Comment:
Technically, you don't need the `slice` here since the destructuring will omit the rest of the array